src/main/java/io/boomerang/mongo/entity/WorkFlowRevisionCount.java

```diff
@@ -0,0 +1,46 @@
+package io.boomerang.mongo.entity;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(Include.NON_NULL)
+@Document(collection = "#{@mongoConfiguration.fullCollectionName('workflows_revisions')}")
```

Comment:
@shenqinb-star if this isn't intended to be a collection, you should remove 

`@Document(collection = "#{@mongoConfiguration.fullCollectionName('workflows_revisions')}")`

and move this out of `src/main/java/io/boomerang/mongo/entity/`

Reply:
OK, make sense

Reply:
@amhudson hi Adrienne, I have moved it to package `mongo/model`, see commit https://github.com/boomerang-io/flow.service.workflow/pull/231/commits/54e159f685838aca8a0a678199b98cbca8da319b

Reply:
If WorkFlowRevisionCount isn't related to an entity in `../mongo/entity`, it shouldn't be in `../mongo/model`. Perhaps `io/boomerang/model` is more appropriate.

Reply:
Although WorkFlowRevisionCount is not related to a collection entity, but it is another kind of representatives for the result of aggregation query which I think it should be prat of mongo.
BTW What's the differences between `../mongo/entity` and `../mongo/model`? I saw some similar models in package `../mongo/model`,

Reply:
1.  `.../mongo/entity/...` and  `.../mongo/model/...` establish the structure of the database records. `.../mongo/entity/...` contains the structure of the database collections while `.../mongo/model/...` are the classes used to support the collections. 

Although `WorkFlowRevisionCount` presents data based on `RevisionEntity`, it is not a mongo model or entity and should not be included in `.../mongo/...`. 

2. The queries defined in `.../mongo/service/...` and `.../mongo/repository/...` are ok as they return data based on `RevisionEntity`.

Reply:
Hey @amhudson as this is _kind of_ a mongo aggregation of sorts, could we stick it in /mongo/model as its an abstraction on a mongo entity but also quite coupled to the mongo entity.

Reply:
Also, @amhudson  great pick up on the annotations!